boilerplates/fe-angular/README.md

```diff
@@ -0,0 +1,80 @@
+# fe-angular
+
+Angular quick starter project
+
+ ## Purpose of this quickstarter
```

Comment:
If you remove the leading space, then Github diff will show some highlighting :)